src/Symfony/Component/Form/Resources/translations/validators.hu.xlf

```diff
@@ -18,6 +18,122 @@
+                <target>A rejtett mező érvénytelen.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Adjon meg egy egész számot.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Kérjük, válasszon érvényes nyelvet.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Kérjük, válasszon érvényes területi beállítást.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Kérjük, adjon meg érvényes pénzösszeget.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Írjon be egy számot.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>A jelszó érvénytelen.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Adjon meg egy százalékos értéket.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Az értékek nem egyeznek.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Adjon meg egy érvényes időpontot.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Kérjük, válasszon érvényes időzónát.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Kérem, írjon be egy érvényes url-t.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Kérjük, adjon meg egy érvényes keresési kifejezést.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>Please provide a valid phone number.</source>
+                <target>Kérjük, adjon meg érvényes telefonszámot.</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>A jelölőnégyzet értéke érvénytelen.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Kérjük valós e-mail címet adjon meg!</target>
```

Comment:
Maybe no shouting =)

Let's remove the exclamation mark (!).

Reply:
Ahh that things, i'll remove the (!)